UC6.1 next revision

eric-schuh opened the pull request:
Mostly cleanup on this one.

![image](https://user-images.githubusercontent.com/22247137/177390412-2fa83e84-e461-4930-91ea-cca173f01012.png)


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/eric-schuh/vc-api-use-cases/pull/16.html" title="Last updated on Nov 22, 2022, 7:18 PM UTC (a14a57a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api-use-cases/16/7b6efca...eric-schuh:a14a57a.html" title="Last updated on Nov 22, 2022, 7:18 PM UTC (a14a57a)">Diff</a>

Commit:
new mermaid for UC6.1

Commit:
adding new sequence diagram mermaid for UC6.1

eric-schuh committed:
Merge branch 'main' into UC6-1_new_rev

eric-schuh commented:
2011-7-19 Call - Need to figure out how to harmonize 22-30 w/CHAPI and clarify what is happening here. How do we turn this into a generic query?

TallTed commented on index.html:
```diff
@@ -219,53 +217,53 @@ <h3>Get Digital Permanent Resident Card</h3>
-            authn.io - Holder Service->>Louis's Browser - Holder App:HolderApp/credentialsStored
+            sequenceDiagram
+              autonumber
+            Lana (Issuer Admin)->>uscis.gov (Issuer App):Configure to properly use Generic Issuer SAAS
```
Does Mermaid allow inline linebreaks (`<br>` or `<br />`)? If so, I strongly suggest putting a linebreak before each parenthetical in these labels, e.g. ---
```suggestion
            Lana<br>(Issuer Admin)->>uscis.gov<br>(Issuer App):Configure to properly use Generic Issuer SAAS
```
(Space chars around the `<br>`s would help readability of these Mermaid source blocks, if it can also handle these.)
```suggestion
            Lana <br> (Issuer Admin)->>uscis.gov <br> (Issuer App):Configure to properly use Generic Issuer SAAS
```
I'm happy to do these PRs, mind numbing as they would be, because I think they'll increase diagram clarity (and possibly allow the tools to increase font sizes, due to less width requirement for the text, if I understand how Mermaid does its job).

Note -- Neither of the suggestions in this comment can be accepted via GitHub, because it can only handle one suggestion per comment. That's fine, as they are insufficient on their own, only presented here as visual aids, and will be trivial to roll into the larger changes I'm really suggesting, if executing this suggestion is viable.

TallTed replied:
The sample "Sequence diagram" a bit below [Diagram Types](https://mermaid-js.github.io/mermaid/#/?id=diagram-types) includes —
```
    Note right of John: Rational thoughts <br/>prevail!
```
— so it does support the clarity-improving `<br/>`s I suggested. :-)

eric-schuh replied:
I think lets wait until I get my script updates that should handle the <br/> issue. Since we are hoping to do more of this kind of work in the future I would like to get a non-manual way of getting these formatting issues resolved, because as you said it would be very tedious to do manually for all the diagrams.

eric-schuh replied:
To give a quick update on this. I have the script that I am using to export the mermaid updated to include the <br> changes. Joe and I are holding off on accepting these PRs to wait for some work unifying the URL structure across the documents.

TallTed commented on index.html:
```diff
@@ -386,15 +384,21 @@ <h3>Execute Multi-stage Presentation Workflow</h3>
-    <p>Tod is using digital credentials to apply for a service/qualify for a job/gain access to a programme. 
+    <p>Kenzie is using digital credentials to apply for a job at as a teacher. 
+      Her teaching certification is already in her digital wallet but the school also requires a 
+      criminal background check which Kenzie does not currently have a credential for. The process
```
```suggestion
      criminal background check for which Kenzie does not currently have a credential. The process
```

TallTed commented on index.html:
```diff
@@ -386,15 +384,21 @@ <h3>Execute Multi-stage Presentation Workflow</h3>
+    <p>Kenzie is using digital credentials to apply for a job at as a teacher. 
+      Her teaching certification is already in her digital wallet but the school also requires a 
+      criminal background check which Kenzie does not currently have a credential for. The process
+      of getting the background check can take 24-48 hours. Kenzie goes to the job application
```
```suggestion
      of getting the background check can take 24&ndash;48 hours. Kenzie goes to the job application
```

TallTed commented on index.html:
```diff
@@ -386,15 +384,21 @@ <h3>Execute Multi-stage Presentation Workflow</h3>
+      Her teaching certification is already in her digital wallet but the school also requires a 
+      criminal background check which Kenzie does not currently have a credential for. The process
+      of getting the background check can take 24-48 hours. Kenzie goes to the job application
+      website and submits her application, the website responds requesting both her teaching credential
```
```suggestion
      website and submits her application; the website responds requesting both her teaching credential
```

eric-schuh replied:
@TallTed Thanks for all the formatting suggestions and grammar clarifications. Just wanted you to know that they should be resolved in the next week, I am working on some updates to our script that outputs the mermaid to include some of them automatically so (e.g. the <br> tags) so I don't have to add those manually in the future.

TallTed replied:
Thanks for the update, @eric-schuh -- and glad I didn't leap into adding all the `<br>` tags! I hadn't absorbed that the mermaid was being generated somewhere, rather than being a major hand-craft. Might be a good thing to add a notice of this before and/or after the mermaid blocks in the documents, so nobody else edits them directly!

eric-schuh committed:
Update index.html

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

eric-schuh committed:
Update index.html

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

eric-schuh committed:
Update index.html

Co-authored-by: Ted Thibodeau Jr <tthibodeau@openlinksw.com>

eric-schuh committed:
Adding <br/> to make lifeline names clearer

eric-schuh committed:
remove / from <br/>

trying to see if this lets rendering work

eric-schuh commented:
@TallTed it seems that while Mermaid supports the `<br>` tag that our renderer here does not (see preview). I open an issue on the respec-mermaid repo https://github.com/digitalbazaar/respec-mermaid/issues/4 to see if we can get support for `<br>` tags

TallTed commented:
@eric-schuh -- Yes, as noted on #22.  Thank you for logging the issue to https://github.com/digitalbazaar/respec-mermaid/issues/4; I'll track that, too.

(Note that you need to code fence the `<br />` within GitHub comment bodies; else, it's treated as HTML markup and just breaks the line)

eric-schuh commented:
Thanks for the catch of the code blocks. Forgot those.

eric-schuh committed:
remove br tags

Removing the "br" tags for line breaks, currently these break the mermaid rendering
